### PR TITLE
feat: upgrade to ng 18

### PR DESCRIPTION
**Pull request by hoebbelsB**

### CODE REVIEW

File: libs/template/package.json [hunk 1]
```diff
@@ -43,9 +43,9 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": "^17.0.0",
+    "@angular/core": "^18.0.0",
     "rxjs": "^6.5.3 || ^7.4.0",
```
Comment (edbzn) [inline libs/template/package.json:47]: I can't find the information on the website but I'm wondering if Angular v18 changed its compat range with RxJS?

**Review by edbzn — CHANGES_REQUESTED** [review]
The commit message should include the breaking change in order to follow semver. The rest looks good!

Edit: The new dependency `@angular/build` seems to be missing.

File: libs/template/package.json (hunk 1 above)
Comment (hoebbelsB) [reply libs/template/package.json:47]: doesn't look like it https://github.com/angular/angular/blob/main/packages/core/package.json

**Comment by JeanMeche** [comment]
`@angular-devkit/build-angular` has `@angular/build` as dependency, so it shouldn't be an issue, right ?

**Pull request merged**